<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameCalendar</class>
 <widget class="QFrame" name="FrameCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MainVLayout">
    <item>
     <layout class="QVBoxLayout" name="VLayout1">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>338</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="HLayoutChoixDate">
          <item>
           <widget class="QComboBox" name="comboMois"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboYear"/>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonPrevious">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonNext">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="VLayout2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
